100+ Python challenging programming exercises.txt

```diff
@@ -164,8 +164,7 @@ import math
 c=50
 h=30
 value = []
-items=[x for x in raw_input().split(',')]
-for d in items:
+for d in raw_input().split(','):
```

Comment:
value = [str(int(round(math.sqrt(2*c*float(d)/h)))) for d in raw_input().split(',')]